homework_02/petar_damyanov_17/views.py

```diff
@@ -0,0 +1,76 @@
+from django.shortcuts import render
+import requests, datetime, pytz
+
+def train_data(request):
+    massachusetsTime = datetime.datetime.now(pytz.timezone('US/Eastern'))
+    
+    params = {
+        'page[offset]': '0',
+        'page[limit]': '50',
+        'include': 'trip,schedule,stop',
+        "sort" : 'departure_time',
+        'filter[stop]': 'place-north',
+        'filter[route_type]': '2',
+        'filter[direction_id]': '0',
+    }
+
+    response = requests.get('https://api-v3.mbta.com/predictions', params=params)
+    json_response = response.json()
+
+    trips = [] 
```

Comment:
It is good that you store the information in one object (one list in the case here), not in multiple ones.